### PR TITLE
CLOUDP-264334: Explicitly trust ubi certs in Dockerfile

### PR DESCRIPTION
**Pull request by josvazg**
### All Submissions:

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

**Review by helderjs — APPROVED** [review]

**Comment by filipcirtog** [comment]
This is a nicer approach. It's better not to rely on Go for handling Ubuntu certificates.

**Review by filipcirtog — APPROVED** [review]
Nice additions with the registry refactoring!

**Comment by igor-karpukhin** [comment]
Was it tested for certification?

**Comment by josvazg** [comment]
> Was it tested for certification?

Yes, see:
https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/10145718181/job/28052419083?pr=1721
Step: `Do preflight-check on test image`

File: .github/workflows/test-e2e.yml [hunk 1]
```diff
@@ -56,12 +56,23 @@ jobs:
...
+        uses: ./.github/actions/certify-openshift-images
+        with:
+          registry: ghcr.io
+          registry_password: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ env.REPOSITORY }}
+          version: ${{ steps.prepare.outputs.tag }}
+          rhcc_token: ${{ secrets.RH_CERTIFICATION_PYXIS_API_TOKEN }}
+          rhcc_project: ${{ secrets.RH_CERTIFICATION_OSPID }}
+          submit: false
+
```
Comment (josvazg) [inline .github/workflows/test-e2e.yml:75]: cc @igor-karpukhin ^see here the added self test code running the preflight action

Comment (igor-karpukhin) [reply .github/workflows/test-e2e.yml:75]: Nice! LGTM

**Pull request merged**